src/media_queryable.js

```diff
@@ -6,8 +6,10 @@ module.exports = React.createClass({
   displayName: 'MediaQueryable',
   mediaListener: null,
   propTypes: {
+    className: React.PropTypes.string,
     defaultMediaQuery: React.PropTypes.string.isRequired,
     mediaQueries: React.PropTypes.object.isRequired,
+    style: React.PropTypes.object
```

Comment:
Hate to nitpick on a project w/o eslint, but could you add the trailing comma to match the existing style please? Thanks!